check_run/check_run/doctype/check_run/check_run.json

```diff
@@ -143,13 +144,19 @@
    "options": "Draft\nSubmitted\nConfirm Print\nPrinted"
   },
   {
+   "default": "<span></span>",
```

Comment:
@johnhkelley1 Does this still need to be here?

Reply:
Removed the span. I think <div id='check-run-vue'></div> needs to be injected via JS because I'm destroying it on table mount, otherwise old table data sticks around on document change